dandiapi/api/views/asset.py

```diff
@@ -511,14 +512,20 @@ def destroy(self, request, versions__dandiset__pk, versions__version, **kwargs):
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
 
-    @swagger_auto_schema(query_serializer=AssetListSerializer)
+    @swagger_auto_schema(query_serializer=AssetListSerializer, responses={200: AssetSerializer()})
     def list(self, request, *args, **kwargs):
         serializer = AssetListSerializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
 
-        queryset = self.filter_queryset(self.get_queryset())
-        glob_pattern: str | None = serializer.validated_data.get('glob')
+        # Don't include metadata field if not asked for
+        queryset: QuerySet[Asset] = self.filter_queryset(
+            self.get_queryset().select_related('blob', 'embargoed_blob', 'zarr')
+        )
```

Comment:
```suggestion
        queryset: QuerySet[Asset] = self.filter_queryset(
            self.get_queryset().select_related('blob', 'embargoed_blob', 'zarr')
        )
        # Don't include metadata field if not asked for
```
since `queryset` construction IMHO has nothing to do with that comment.